src/simplify_algebra.cpp

```diff
@@ -851,6 +853,92 @@ struct find_div_const
     }
 };
 
+struct find_zero_div_const
+{
+    auto matcher() const { return match::name("div")(match::arg(1)(match::has_value(0.0f))); }
+
+    void apply [[noreturn]] (const module& m, const match::matcher_result& r) const
```

Comment:
why does it require `[[noreturn]]` flag when return type is `void` ?

Reply:
I think I need to remove the const here, not sure how that slipped in here. I'm also going to flip these params to [[maybe_unused]] instead of void incase that's causing issues since that appears to be more standard to the standard

https://stackoverflow.com/a/44106902